DataReader.py

```diff
@@ -0,0 +1,99 @@
+import datetime
+import csv
+import glob
+import collections
+
+from Weather import Weather
+from View import View
+
+
+class data_reader:
+
+	annual_report = {}
+	annual_max_temp = {}
+	year = ""
+
+
+	def get_max(self, max_value, to_compare):
+		
+		if max_value < to_compare:
+			return True
+		
+		return False
+
+	def get_min(self, min_value, to_compare):
+		
+		if min_value < to_compare and min_value > 0:
+			return min_value
+		
+		return to_compare
+
+	def retreive_dir(self, dir_path):
```

Comment:
Use appropriate name for function that explains what a function does. Function should do only one thing.